Assets/Plugins/CountlySDK/Models/CountlyConfiguration.cs

```diff
@@ -10,31 +10,99 @@ namespace Plugins.CountlySDK.Models
     [Serializable]
     public class CountlyConfiguration
     {
+        /// <summary>
+        /// URL of the Countly server to submit data to.
+        /// Mandatory field.
+        /// </summary>
         public string ServerUrl = null;
+
+        /// <summary>
+        /// App key for the application being tracked.
+        /// Mandatory field.
+        /// </summary>
         public string AppKey = null;
+
+        /// <summary>
+        /// Unique ID for the device the app is running on.
+        /// </summary>
         public string DeviceId = null;
 
+        /// <summary>
+        /// Set to prevent parameter tampering.
+        /// </summary>
         public string Salt = null;
+
+        /// <summary>
+        /// Set to send first app launch segment.
+        /// </summary>
         public bool EnableFirstAppLaunchSegment = false;
+
+        /// <summary>
+        /// Set to send all requests made to the Countly server using HTTP POST.
+        /// </summary>
         public bool EnablePost = false;
+
+        /// <summary>
+        /// Set to true when you don't want to send requests to the Countly server.

```

Comment:
This description leaves the wrong impression. Change it to:
"Set to true if you want the SDK to pretend that it's functioning."